modules/ROOT/pages/managing-api-clients.adoc

```diff
@@ -3,19 +3,37 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This topic provides an overview of client management on Anypoint Platform. As an API owner, you can apply an OAuth 2.0 policy to authenticate client applications that try to access your API. You need an OAuth provider to use an OAuth 2.0 policy. One of the following OAuth providers is recommended:
+This topic provides an overview of client management on Anypoint Platform. As an API owner, you can apply an OAuth 2.0 policy to authenticate client applications that try to access your API. You need an OAuth 2.0 provider to use an OAuth 2.0 policy.
 
-* OpenAM version: 14
-* PingFederate versions: 6 - 8
-* Dynamic Client Registration-compliant identity providers
+You can configure more than one client provider and xref:environments.adoc#add-an-idp-to-an-existing-environment[associate the client providers with different environments].
+When you delete a client provider from your master organization, the client provider is no longer available in environments that used it. Also, assets that used the client provider can no longer authenticate users who want to access them.
+
+You can configure the following OAuth providers:
+
+* xref:conf-client-mgmt-openam-task.adoc[OpenAM version: 14]
+* xref:conf-client-mgmt-pf-task.adoc[PingFederate versions: 6 - 8]
+* xref:configure-client-management-openid-task.adoc[Dynamic Client Registration-compliant identity providers]
 +
-This provider needs to conform to the OpenID Connect Client Registration 1.0 incorporating errata set 1.
+This provider needs to conform to the https://openid.net/specs/openid-connect-registration-1_0.html[OpenID Connect Client Registration 1.0 incorporating errata set 1].
```

Comment:
To what does "this provider" refer: the previous list item?
needs to > must

Reply:
```suggestion
Dynamic Client Registration-compliant identity providers must conform to the https://openid.net/specs/openid-connect-registration-1_0.html[OpenID Connect Client Registration 1.0 incorporating errata set 1].
```